test/request-response-samples/execFile__valid_xml_generated_on_request_with_complex_type_element_form_default_qualified/soap.wsdl

```diff
@@ -11,39 +23,40 @@
 					<element name="startDetached" type="xsd:boolean" minOccurs="1" maxOccurs="1"/>
 				</sequence>
 			</complexType>
+			<element name="execFileResult" type="tns:execFileResult"/>
 			<complexType name="execFileResult">
 				<sequence>
 					<element name="base64EncodedStdOut" type="xsd:string" minOccurs="1" maxOccurs="1"/>
 				</sequence>
 			</complexType>
 		</schema>
 	</types>
-	<message name="execFile">
-		<part name="params" type="RpcExample:execFileParams"/>
+	<message name="execFileRequest">
+		<part name="execFileRequest" element="FileExecutor:execFileParams"/>
 	</message>
 	<message name="execFileResponse">
-		<part name="result" type="RpcExample:execFileResult"/>
+		<part name="execFileResponse" element="FileExecutor:execFileResult"/>
 	</message>
-	<portType name="RpcExamplePortType">
+	<portType name="FileExecutorPortType">
 		<operation name="execFile">
-			<input message="tns:execFile"/>
+			<input message="tns:execFileRequest"/>
 			<output message="tns:execFileResponse"/>
 		</operation>
 	</portType>
-	<binding name="RpcExample" type="tns:RpcExamplePortType">
-		<SOAP:binding style="rpc" transport="http://schemas.xmlsoap.org/soap/http"/>
+	<binding name="FileExecutorBinding" type="tns:FileExecutorPortType">
+		<SOAP:binding style="document" transport="http://schemas.xmlsoap.org/soap/http"/>
 		<operation name="execFile">
-			<SOAP:operation style="rpc"/>
+			<SOAP:operation type="document"/>
 			<input>
-				<SOAP:body use="encoded" namespace="urn:RpcExample" encodingStyle="http://www.w3.org/2003/05/soap-encoding"/>
+				<SOAP:body use="literal"/>
 			</input>
 			<output>
-				<SOAP:body use="encoded" namespace="urn:RpcExample" encodingStyle="http://www.w3.org/2003/05/soap-encoding"/>
+				<SOAP:body use="literal"/>
 			</output>
 		</operation>
 	</binding>
-	<service name="RpcExample">
-		<port name="RpcExample" binding="tns:RpcExample">
+	<service name="FileExecutorService">
```

Comment:
If I don't rename it, the name "RpcExample" will confuse other maintainers -- the WSDL isn't in RPC style any more. Another reason to rename is that the service name, binding name and port name need to be different, at least in some WSDL validator they're required to be different, e.g. https://www.wsdl-analyzer.com/. Before I renamed, they're the same "RpcExample".